File: 1598-crawler-log-folder/1598-crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int count=0;
        for(auto log : logs)
        {
            if(log=="../")
            {
               if(count>0) count-=1;
            }
            else if(log=="./")
            {
                continue;
            }
            else
            {
                count+=1;
            }
        }
        return count;
    }
};

File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        count=0
        for i in logs:
            if(i=="../"):
                if(count>0):
                    count=count-1
            elif(i=="./"):
                continue
            else:
                count+=1;
        return count